jib-core/src/main/java/com/google/cloud/tools/jib/hash/CountingDigestOutputStream.java

```diff
@@ -54,10 +48,13 @@ public CountingDigestOutputStream(OutputStream outputStream) {
   }
 
   /**
-   * Computes the hash and remembers the size of the bytes written to compute the hash. The buffer
-   * resets after this method is called, so this method should only be called once per computation.
+   * Computes the hash and returns it along with the size of the bytes written to compute the hash.
+   * The buffer resets after this method is called, so this method should only be called once per
+   * computation.
```

Comment:
I wonder how many places do we call this method multiple times?

Reply:
I'm guessing none, because the last time I touched this class, it was reporting wrong bytes when called more than once. (Basically, `bytesSoFar` was not reset.)